dotnet/samples/AgentWebChat/AgentWebChat.AgentHost/Program.cs

```diff
@@ -78,8 +78,20 @@ Once the user has deduced what type (knight or knave) both Alice and Bob are, te
     description: "An agent that helps with literature.",
     chatClientServiceKey: "chat-model");
 
-builder.AddSequentialWorkflow("science-sequential-workflow", [chemistryAgent, mathsAgent, literatureAgent]).AddAsAIAgent();
-builder.AddConcurrentWorkflow("science-concurrent-workflow", [chemistryAgent, mathsAgent, literatureAgent]).AddAsAIAgent();
+var scienceSequentialWorkflow = builder.AddWorkflow("science-sequential-workflow", (sp, key) =>
+{
+    List<IHostedAgentBuilder> usedAgents = [chemistryAgent, mathsAgent, literatureAgent];
```

Comment:
FWIW, it's still counterintuitive to me that something called "Agent" and that's returned from a method called "AddAIAgent" is actually a builder. I understand why it is, but maybe there are things we could do to ease this.